crates/astria-bridge-withdrawer/local.env.example

```diff
@@ -37,9 +37,6 @@ ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_ADDRESS_PREFIX=astria
 # The fee asset denomination to use for the bridge account's transactions.
 ASTRIA_BRIDGE_WITHDRAWER_FEE_ASSET_DENOMINATION="nria"
 
-# The minimum expected balance of the fee asset in the bridge account.
-ASTRIA_BRIDGE_WITHDRAWER_MIN_EXPECTED_FEE_ASSET_BALANCE=1000000
```

Comment:
Removing this is a breaking change but it's not marked in the PR title nor explained in the PR message.

Reply:
it is mentioned in the PR and there was an issue for it that I forgot to link.

The balance check was unnecessary and removing it resolved some of your previous comments regarding exponential backoffs and serialization errors.